src/pages/iou/request/step/IOURequestStepParticipants.tsx

```diff
@@ -221,7 +221,7 @@ function IOURequestStepParticipants({
 
             // When multiple participants are selected, the reportID is generated at the end of the confirmation step.
             // So we are resetting selectedReportID ref to the reportID coming from params.
-            if (val.length !== 1 && !isInvoice) {
+            if (val.filter((item) => !!item.login).length !== 1 && !isInvoice) {
```

Comment:
Looking good!
Could you please add a short comment explaining why this logic is needed? It's not clear at first glance